resources/views/layouts/app.blade.php

```diff
@@ -18,6 +18,9 @@
 
     <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <link href="{{ asset('css/bootstrap.css') }}" rel="bootstrap">
+    <link rel="stylesheet" href="{{asset('css/bootstrap-select.css')}}">
+
```

Comment:
These lines were not here if you would have rebased your branch on top of the master branch.

Reply:
This  is mistake please ignore it.